2/src/js/showSectionFunc.js

```diff
@@ -0,0 +1,21 @@
+export const showSectionFunc = () => {
+  const sections = document.querySelectorAll("main > section"),
+    breakpointsDesktop = [600, 1200, 2000, 3000, 3500, 3800],
+    breakpointsTablet = [300, 1200, 1900, 3100, 3700, 4100];
+  let currentBreakpoints =
+    window.innerWidth <= 1024 ? breakpointsTablet : breakpointsDesktop;
+
+  sections.forEach((section, index) => {
+    if (window.pageYOffset >= currentBreakpoints[index]) {
+      if (window.innerWidth <= 1024 && index === 0) {
+        section.style.opacity = "1";
+        document.getElementById("textWrapper").style.opacity = "1";
```

Comment:
I would recomment to avoid declare styles inside js. Create a helper css class and add or remove from element when needed

Reply:
done

Reply:
done